pkg/operator/starter.go

```diff
@@ -604,12 +604,25 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		return err
 	}
 
+	operatorConfig, err := operatorCtx.configClient.ConfigV1().APIServers().Get(ctx, "cluster", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// TODO(4.7): switch over to secure access-token logging by default and delete old non-sha256 tokens
+	auditPolicyPathGetterWithAccessTokenLogs := func(profile string) (string, error) {
+		if operatorConfig.Annotations["oauth-apiserver.openshift.io/secure-token-storage"] == "true" {
```

Comment:
ok - this will be on in 4.6

Reply:
on a downgrade, we will switch back to a different profile.

Reply:
good point.

Reply:
fixed. Using a lister now.